src/sas/qtgui/Perspectives/Invariant/InvariantPerspective.py

```diff
@@ -331,8 +326,11 @@ def calculateThread(self, extrapolation):
             surface = None
 
         if (calculation_failed):
+            self.cmdStatus.setEnabled(False)
             logging.warning('Calculation failed: {}'.format(msg))
             return self.model
+        else:
```

Comment:
This `else` is redundant, since the above condition exits the method. Remove `else` and reset the indentation on line 333 (enabling cmdStatus)

Reply:
hummm... don't remember doing anything with cmdStatus but agreed. Will do.

Reply:
This code was marked as changed/added in this PR, which is why I looked at it :) This is just a stylistic issue, so feel free to just click on `Resolve`.

Reply:
and easy to fix .. I see that Paul K. has spotted another instance (mine in that case) and labelled "lint" I will fix both shortly